students/shawn_degraw/session02/series.py

```diff
@@ -0,0 +1,72 @@
+def fibonacci(n):
+    """
+    compute the nth Fibonacci number 
+    
+    :param n: value of the nth fibonacci element
+    """
+    return sum_series(n)
+
+
+def lucas(n):
+    """
+    compute the nth Lucas number
+    
+    :param n: value of the nth lucas element
+    """
+    return sum_series(n, 2, 1)
+
+def sum_series(n, n0=0, n1=1):
```

Comment:
Your sum_series function only works for parameters n0 and n1 that are set equal to those of fibonacci or lucas series.
The idea is to have this function as a more generalized form of fibonacci and lucas and make it universally valid.